### DISCUSSION TITLE
Can I disable a certain action without using the setInteractive action of useVueFlow?

### DISCUSSION THREAD

**Discussion by starter-dev** [bcakmakoglu/vue-flow#693] [Q&A] [answered]
For example, I want to disable the dragging of nodes without locking all the other actions. Is it possible? Thank you!

**Comment by bcakmakoglu** [comment, marked as answer]
Sure, you can use the `nodesDraggable` option to disable/enable dragging for all nodes at once.

the `setInteractive` option really just does this:

```ts
const setInteractive: Actions['setInteractive'] = (isInteractive) => {
    state.nodesDraggable = isInteractive
    state.nodesConnectable = isInteractive
    state.elementsSelectable = isInteractive
  }
```

You can set all of these options separately yourself, no need to use the `setInteractive` action.

**Reply by starter-dev** [reply]
Got it! That's exactly what I intend to do. Thank you so much!
